test/integration/host_test.rb

```diff
@@ -118,12 +118,11 @@ def class_params
       visit new_host_path
 
       select2 hg1.name, :from => 'host_hostgroup_id'
-
       wait_for_ajax
 
       find("#host_environment_id + .input-group-btn .btn").click
```

Comment:
I could reproduce it a bit more frequently (1/4 times) by setting a `sleep 2` before the find.